sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminAsyncClient.java

```diff
@@ -0,0 +1,370 @@
+    public Mono<Evaluation> getEvaluation(String evaluationId) {
+        return getEvaluationWithResponse(evaluationId).flatMap(FluxUtil::toMono);
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Evaluation>> getEvaluationWithResponse(String evaluationId) {
+        try {
+            return withContext(context -> getEvaluationWithResponse(evaluationId, context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    Mono<Response<Evaluation>> getEvaluationWithResponse(String evaluationId, Context context) {
+        if (CoreUtils.isNullOrEmpty(evaluationId)) {
+            throw logger.logExceptionAsError(new IllegalArgumentException("'evaluationId' is required and cannot be null or empty"));
+        }
+        return service.getEvaluations().getWithResponseAsync(evaluationId, context)
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(response -> new SimpleResponse<>(response, response.getValue()));
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> deleteEvaluation(String evaluationId) {
+        return deleteEvaluationWithResponse(evaluationId).flatMap(FluxUtil::toMono);
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Void>> deleteEvaluationWithResponse(String evaluationId) {
+        try {
+            return withContext(context -> deleteEvaluationWithResponse(evaluationId, context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    Mono<Response<Void>> deleteEvaluationWithResponse(String evaluationId, Context context) {
+        if (CoreUtils.isNullOrEmpty(evaluationId)) {
+            throw logger.logExceptionAsError(new IllegalArgumentException("'evaluationId' is required and cannot be null or empty"));
+        }
+        return service.getEvaluations().deleteWithResponseAsync(evaluationId, context)
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(response -> new SimpleResponse<>(response, null));
+    }
+
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public Mono<List<Evaluation>> getEvaluations() {
+        return getEvaluationsWithResponse().flatMap(FluxUtil::toMono);
+    }
+
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public Mono<Response<List<Evaluation>>> getEvaluationsWithResponse() {
+        try {
+            return withContext(context -> getEvaluationsWithResponse(context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    Mono<Response<List<Evaluation>>> getEvaluationsWithResponse(Context context) {
+        return service.getEvaluations().listWithResponseAsync(context)
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(response -> new SimpleResponse<>(response, response.getValue()));
```

Comment:
Since the models are all now in the models folder (there is no implemenation/models), this is not relevant any more. But yeah .. we are returning the model directly returned by autorest.